src/data_augmentations/gaussian.py

```diff
@@ -0,0 +1,36 @@
+from typing import Tuple
+import numpy as np
+
+
+def augment_data(X: np.ndarray, Y: np.ndarray, noise: float = 0.05, iterations: int = 1) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
I would rename `iterations` as `n_augmentations` or something like that, more explicit.

Reply:
fixed in #7